apps/explorer/components/LoadingTimestamp.tsx

```diff
@@ -1,12 +1,22 @@
 import { ClientSideOnly, Skeleton } from '@siafoundation/design-system'
+import clsx from 'clsx'
 
 export default function LoadingTimestamp({
   children,
+  width,
 }: {
   children: React.ReactNode
+  width?: string
 }) {
   return (
-    <ClientSideOnly fallback={<Skeleton className="w-[200px] h-[20px]" />}>
+    <ClientSideOnly
+      fallback={
+        <Skeleton
+          className={clsx('h-[20px]', width ? `w-[${width}]` : 'w-[200px]')}
```

Comment:
tailwind scans to codebase to understand what classes to generate and inject in the resulting css, it will find any static class names in the codebase but it will not be able to generate a class for string variable concatenation like `w-[${width}]`. It will work if you add a `className` prop and pass the full `w-[130px]`.